### PR TITLE
add setting and config for custom template

### LINKED ISSUE #300: Create custom map report templates with INGC brand
Do English re-design before #299 and then translate that into PT in #299. 

Use the design outputs from https://github.com/kartoza/geonode/issues/420

child of #306

### PR DESCRIPTION
**Pull request by myarjunar**
fix kartoza/geosafe#300 
required by https://github.com/kartoza/geosafe/pull/395
required by https://github.com/kartoza/ingc_geonode_theme/pull/4

### CODE REVIEW

File: src/core/settings.py [hunk 1]
```diff
@@ -112,6 +112,12 @@ def update_settings_from(settings_name):
 
         TEMPLATES[0]['DIRS'] = template_dirs
 
+        # The path is {custom_theme_path}/templates/geosafe/qgis_templates
+        QGIS_REPORT_TEMPLATE_DIRECTORY = os.path.join(
```
Comment (lucernae) [inline src/core/settings.py]: Please put this settings on core/geosafe.py ;) @myarjunar

Comment (lucernae) [inline src/core/settings.py]: Please put this settings on core/geosafe.py ;) @myarjunar

**Review by lucernae — APPROVED** [review]
LGTM! Thanks @myarjunar  with just a comment.

File: src/core/settings.py (hunk 1 above)
Comment (lucernae) [inline src/core/settings.py]: Please move this to core/geosafe.py ;) @myarjunar

Comment (lucernae) [inline src/core/settings.py]: Please move this to core/geosafe.py ;) @myarjunar

**Review by lucernae — APPROVED** [review]
LGTM @myarjunar , Thanks! with just a comment.

**Review by boney-bun — APPROVED** [review]
LGTM

**Pull request merged**